cmake/FindCharm.cmake

```diff
@@ -54,13 +54,13 @@ find_library(CHARM_LIBCK
     HINTS ${CHARM_ROOT} ENV CHARM_ROOT
 )
 find_library(CHARM_EVERYLB
-    NAMES libmoduleEveryLB.a
+    NAMES moduleEveryLB
```

Comment:
Oh we should have caught this when reviewing #3077, sorry.

I think you should also add `NO_DEFAULT_PATH`, replace `CHARM_ROOT` with `CHARM_LIBRARIES` and move the `find_library` call below the definition of `CHARM_LIBRARIES` below. My intention is that the library is only searched in the libs directory that is actually linked.

Reply:
makes sense; squashed in the changes.